this_the_way.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+
+def rename_files(path):
+    list_of_file = os.listdir(path)
+    for entry in list_of_file:
```

Comment:
1. Is there a way to use list comprehension here instead of the for loop?
2. We need to return a list that contains the names of all of the files that start with "deep", instead of printing the names.